src/frontend/komponenter/ServiceContext.tsx

```diff
@@ -46,9 +53,20 @@ const [ServiceProvider, useServiceContext] = constate(() => {
         oppdaterValgtService(services, pathname);
     }, [pathname]);
 
+    useEffect(() => {
+        if (services.status === RessursStatus.SUKSESS) {
+            services.data.map((service) => {
+                hentTaskerSomHarFeilerEllerErTilManuellOppføling(service).then((response) => {
```

Comment:
Nå henter vi data for både gammel og ny variant hver gang. Kunne vi hentet denne først, og dersom noe feiler - sette en state som gjør at vi henter på "gammel" måte for de servicene som IKKE har fått på den nye formen?

Reply:
(Litt massivt med 16 kall for å hente status)